.github/workflows/build.yml

```
@@ -20,6 +23,7 @@ jobs:
         uses: actions/setup-node@v4
         with:
           node-version: "20"
+          node-version: "20"
```

actionlint (1.7.8)

key "node-version" is duplicated in "with" section. previously defined at line:25,col:11. note that this key is case insensitive

(syntax-check)